tools/msp_blastparser_and_hits/BlastParser_and_hits.xml

```diff
@@ -1,12 +1,12 @@
 <tool id="BlastParser_and_hits" name="Parse blast output and compile hits" version="2.4.3">
 <description>for virus discovery</description>
 <requirements></requirements>
-<command interpreter="python">
-BlastParser_and_hits.py
-	--sequences $sequences
-	--blast $blast
-	--tabularOutput $tabularOutput
-	--fastaOutput $fastaOutput
+<command><![CDATA[
+    python '$_tool_directory__'/BlastParser_and_hits.py
```

Comment:
i know, and there are a few other things in this wrapper that are not correct.
The [WIP] label means work in progress, I will change this when its done and a review is helpful.